doc/developers-guide/accumulators.md

```diff
@@ -61,6 +71,10 @@ Acc2 = mongoose_acc:add_prop(myprop, 123, Acc1),
 V = mongoose_acc:get_prop(myprop, Acc2).
 ```
 
+The rationale behind stripping an accumulator is that many values store in it are context-dependend.
```

Comment:
The rationale behind stripping an accumulator is that many values stored in it are context-dependend.